pkg/daemon/ceph/client/upgrade.go

```diff
@@ -177,14 +187,6 @@ func OkToStop(context *clusterd.Context, clusterInfo *ClusterInfo, deployment, d
 func OkToContinue(context *clusterd.Context, clusterInfo *ClusterInfo, deployment, daemonType, daemonName string) error {
 	// the mon case is handled directly in the deployment where the mon checks for quorum
 	switch daemonType {
-	case "osd":
-		if osdDoNothing(context, clusterInfo) {
-			return nil
-		}
-		err := okToContinueOSDDaemon(context, clusterInfo)
-		if err != nil {
-			return errors.Wrapf(err, "failed to check if %s was ok to continue", deployment)
-		}
 	case "mds":
 		err := okToContinueMDSDaemon(context, clusterInfo, deployment, daemonType, daemonName)
```

Comment:
@BlaineEXE @leseb Is there still a use for ok-to-continue with any of the daemons? Or does MDS still need it for some reason? Removing ok-to-continue completely could be a separate PR though...

Reply:
I haven't looked at MDS stuff in a while. Looks like Seb was the one who created `okToContinueMDSDaemon`. Maybe he remembers something more about it?

Reply:
The fs is much more critical in terms of availability, waiting for the MDS to become active again should happy pretty quickly. I think it's worth waiting.

Reply:
What is there to wait for?  Is there something left in the reconcile?

Reply:
Wait for the MDS to be active again or in standby-replay.